tools/add_strelka2_fields.cwl

```diff
@@ -13,15 +13,22 @@ requirements:
     coresMin: $(inputs.cores)
   - class: ShellCommandRequirement
 
-baseCommand: [/usr/bin/add_strelka2_fields.py]
+baseCommand: []
+
+arguments:
+  - position: 0
+    shellQuote: false
+    valueFrom: >-
+      $(inputs.run_tool_flag ? ">&2 /usr/bin/add_strelka2_fields.py" : ">&2 echo 'User opted to skip adding fields to vcf' && exit 0;")
 
 inputs:
   strelka2_vcf:
     type: File
     inputBinding:
       position: 1
       prefix: '--strelka2_vcf'
-    secondaryFiles: .tbi
+    secondaryFiles: ['.tbi']
+  run_tool_flag: {type: boolean?, doc: "When run as part of a workflow, can skip", default: false}
```

Comment:
You actually don't need to set the default here because `null` and `false` are treated the same by the `?` operator.

Reply:
I suppose. It makes me paranoid using the two as equivalents